beast/fitting/fit_metrics/common.py

```diff
@@ -119,42 +119,21 @@ def percentile(data, percentiles, weights=None):
 
     _wt = np.asarray(weights, dtype=float)
 
-    if _C_code:
-        return c_wp(_data, _wt, _p)
-    else:
-        len_p = len(_p)
-        sd = np.empty(n, dtype=float)
-        sw = np.empty(n, dtype=float)
-        aw = np.empty(n, dtype=float)
-        o = np.empty(len_p, dtype=float)
-
-        i = np.argsort(_data)
-        np.take(_data, i, axis=0, out=sd)
-        np.take(_wt, i, axis=0, out=sw)
-        np.add.accumulate(sw, out=aw)
-
-        if not aw[-1] > 0:
-            raise ValueError("Nonpositive weight sum")
-
-        w = (aw - 0.5 * sw) / aw[-1]
-
-        spots = np.searchsorted(w, _p)
-        for (pk, s, p) in zip(list(range(len_p)), spots, _p):
-            if s == 0:
-                o[pk] = sd[0]
-            elif s == n:
-                o[pk] = sd[n - 1]
-            else:
-                f1 = (w[s] - p) / (w[s] - w[s - 1])
-                f2 = (p - w[s - 1]) / (w[s] - w[s - 1])
-                if not (
-                    (f1 >= 0) and (f2 >= 0) and (f1 <= 1) and (f2 <= 1)
-                    and (abs(f1 + f2 - 1.0) < 1e-6)
-                ):
-                    raise AssertionError()
-                o[pk] = sd[s - 1] * f1 + sd[s] * f2
-        return o
+    
+    i = np.argsort(_data)
```

Comment:
`i` usually refers to an integer in a loop.  Just for consistency - could you change this to be something more descriptive than `i`?  Like `indxs` or something like that?

Reply:
Also, the if statement to the c code has been removed.   I'm find with this, but isn't there some C code in the repository that needs removing as well then?

Reply:
'i' has been changed to 'isort'. 

At the beginning of the code, you can see '_C_code = False' in line 55. I assume someone made a decision not to use a function from C to compute the weighted percentile. So I removed the if statement inside the function. If we want to leave the option to use the function from C when is available, I can revert the if statement and remove the line 55.